### ISSUE TITLE
Can't remove applied filters form nexus search

### ISSUE THREAD

**Issue by lukamaksic** [Nexus-Mods/web-issues#533]
Bug/error being reported:
I can't remove applied filters form nexus search, no matter what I do. Clearing them or unselecting them them then applying the new selection does not work. To point it out just in case, these filters were applied before the nexus design was changed,

**Comment by iamtiz** [comment]
Hi, do you mean blocked content? You can clear it in your preferences, at https://www.nexusmods.com/users/myaccount?tab=content+blocking

**Comment by lukamaksic** [comment]
No, I was actually talking about the attributes in the refine results options. https://drive.google.com/file/d/1qpSazIn9TuFTMx56gp6LZl1b92DJeeBC/view?usp=sharing
If i recall correctly, I tried filtering out these a few months ago, and now a simply can't remove the selected attributes. I can unselect them, but they are reapplied every time i refresh the page or search for mods again

**Comment by Eolhin** [comment]
Nice to know I am not the only one with stuck attributes.  Though in my case, when I hit clear selection, and it puts the checks back in, it behaves logically like they were unchecked in the search results.  Having more then half the attributes behaving logically like they are the opposite of what they appear to be set to though makes searching extremely difficult.

**Comment by Eolhin** [comment]
How is this an "enhancement" and not a bug?  *sighs*

**Comment by MYSTERlOUSGUY** [comment]
[https://www.nexusmods.com/users/myaccount?tab=content+blocking](https://www.nexusmods.com/users/myaccount?tab=content+blocking)

This is main issue here. I have lots of blocked tags but i can't remove any of them.
I can click on them but nothing happens.
Its no wonder i get 5 search results and 10000 blocked by filters. Please fix this ASAP.
My Account name is MysteriousGuy .

**Comment by Eolhin** [comment]
I am getting the problem with Chrome, if that makes any difference.

**Comment by iamtiz** [comment]
Are you still having this issue?

**Comment by Eolhin** [comment]
Yes, I just checked, and I am still having the problems I have described repeatedly across several different bug reports.

**Comment by Eolhin** [comment]
To be clear, how is Attributes under Refine Results supposed to work?  For instance, with nothing set under Content Blocking, if I put in a search term, and put a check mark in Translation under Attributes, is that supposed to include translations in the results, show only things tagged with translation, or exclude translations from the results?  If Anime is checked, is that supposed to include files tagged with Anime?  And if so, does that mean if it is not checked, then all files tagged with Anime are excluded from the search?  And even if the use of that section was clear, I still have the problem of Clear Selection putting checks in some things, rather then taking them out, and those things defaulting to checked when I conduct a new search.

**Comment by iamtiz** [comment]
The contents in "attributes" are included in results only if they are checked, if they are not checked then they are not included. They are not exclusive checks, so if "anime" is checked, then anime content is included, along with non-anime content. If it's not checked then only non-anime content is included.

The "clear selection" resets the checkbox to their default behaviour, so all checked except for "has images", "nmm compatible" and "only show adult files" - which are exclusive checks (checking them will exclude some content, while the other checks will include some content)

**Comment by Eolhin** [comment]
Ah, you mixed check to include and check to exclude things in the same section, and "clear selection" does not do the same thing to all check boxes...  No wonder I have been having problems.  So, in order to not see Anime content, for instance, I have to open up Refine results and uncheck Anime every single time I do a search?  Because every time I do a new search, or hit clear selection, it defaults to the settings you described, with most of the boxes checked.  I have been able to filter out Translations by adding that tag to 'Content Blocking' for my account.  I used to have 'Anime' under Content Blocking as well, but I can no longer find an option for that there.  (I did check the drop-down list, and have tried selecting both Skyrim games, as well as Global Tags, but that still isn't an option.)

Well, at least I understand how it is supposed to work now, even if that is a really confusing way to do things.  "Clear Selection" should really be "Reset to Include All".
